package.json

```diff
@@ -40,28 +40,29 @@
   "homepage": "https://docs.opensea.io/v1.0/reference",
   "dependencies": {
     "bignumber.js": "^4.0.4",
+    "ethereumjs-abi": "git+https://github.com/ProjectWyvern/ethereumjs-abi.git",
     "ethereumjs-util": "^5.1.3",
     "fbemitter": "^2.1.1",
     "isomorphic-unfetch": "^2.1.1",
     "json-loader": "^0.5.7",
     "lodash": "^4.17.4",
-    "query-string": "^6.1.0",
+    "query-string": "6.11.0",
     "web3": "^0.20.6",
     "webpack": "^3.10.0",
-    "wyvern-js": "github:ProjectOpenSea/wyvern-js#semver:^3.2.0",
-    "wyvern-schemas": "github:ProjectOpenSea/wyvern-schemas#semver:^0.10.7"
+    "wyvern-js": "github:ProjectOpenSea/wyvern-js#semver:^3.2.1",
+    "wyvern-schemas": "github:ProjectOpenSea/wyvern-schemas#semver:^0.11.1"
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run lint && npm test && npm run build"
+      "pre-commit": "npm run lint && npm run build && git add ."
```

Comment:
It would be better to do it this way:
```
  "husky": {
    "hooks": {
      "pre-commit": "npm run lint && npm test && npm run build",
      "post-commit": "git add lib/*"
    }
  },
```
`git add lib/*` won't add all the files in the directory as there could be times we don't want to commit all files.
Also it will only run `git add lib/*` if you complete the commit.